### DISCUSSION TITLE
Not able to open corporate site via cypress in Windows 11

### DISCUSSION THREAD

**Discussion by Astha626** [cypress-io/cypress#21584] [Questions and Help]
Not able to open corporate site via cypress in Windows 11. Tried all the proxy settings available on Google and even configured all the proxy settings given on cypress official site but no luck.  With same setup I'm able to access localhost url as well as other sites which are in public domain. But there is some issue in accessing intranet or private sites. 

**Cypress could not verify that this server is running:

> https://corportesite

This server has been configured as your baseUrl, and tests will likely fail if it is not running.**

**Comment by muratkeremozcan** [comment]
I had to wrestle this a lot in the previous job.

What happens if you use Electron, and the other browsers? If Electron is ok, you might be able to blow up some Chrome registry settings pushed by IT. You probably want a batch file clearing those, or need a special ticket to enable development on your work PC.

Try to match the CI environment if possible... That is the meta.

**Reply by Astha626** [reply]
This warning appears as soon as I run cypress runner like below. And if I still try to run the script then it says: 
cy.visit() failed trying to load:
We attempted to make an http request to this URL but the request failed without a response.
We received this error at the network level:
  > Error: A connection to the upstream proxy could not be established: connect ETIMEDOUT 145.7.175.233:80
Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer
<img width="956" alt="image" src="https://user-images.githubusercontent.com/105936573/169574990-48459556-d412-4268-b435-bad0aa86b118.png">
<img width="917" alt="image" src="https://user-images.githubusercontent.com/105936573/169575570-7987c420-2756-4a76-b2c4-0b79e300f56a.png">

**Reply by mpanchal76** [reply]
Hi There, did you found resolution for this issue.  I am in the same boat...I am having issues opening certain urls.
